src/tests/ftest/util/slurm_utils.py

```diff
@@ -287,20 +292,24 @@ def srun_str(hosts, cmd, srun_params=None):
     return str(cmd)
 
 
-def srun(hosts, cmd, srun_params=None, timeout=30):
+def srun(hosts, cmd, srun_params=None, timeout=60):
     """Run srun cmd on slurm partition.
 
     Args:
         hosts (str): hosts to allocate
         cmd (str): cmdline to execute
         srun_params(dict): additional params for srun
+        timeout
 
     Returns:
         CmdResult: object containing the result (exit status, stdout, etc.) of
             the srun command
 
     """
-    cmd = srun_str(hosts, cmd, srun_params)
+    srun_time = int(timeout/60)
+    if srun_time < 1:
```

Comment:
(pylint-consider-using-max-builtin) Consider using 'srun_time = max(srun_time, 1)' instead of unnecessary if block